### PR TITLE
1_packed

### PR DESCRIPTION
**Pull request by AnaScia**
Add CSS modularity for each component created.

### CODE REVIEW

File: src/styles/table.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  width: 100%;
+  border: 1px solid #000;
+}
+
+th,
+td {
+  width: 25%;
+  text-align: center;
+  border: 1px solid orangered;
+  /* border-collapse: collapse; */
```
Comment (Alvacampos) [inline src/styles/table.css]: Clean this.

File: src/styles/table.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+thead {
+  background-color: lightsalmon;
+}
+
+.icon {
+  display: inline-block;
+  width: 1em;
+  height: 1em;
+  stroke-width: 0;
+  stroke: currentColor;
```
Comment (Alvacampos) [inline src/styles/table.css]: currentColor? what is this?

File: src/styles/table.css (hunk 1 above)
Comment (AnaScia) [reply src/styles/table.css]: Ready

File: src/styles/table.css (hunk 2 above)
Comment (AnaScia) [reply src/styles/table.css]: I deleted it

**Comment by AnaScia** [comment]
Packed had changes like: 
-Edit CSS,I used Flexbox for cell.
-Add button edit for edit text from Item.
image

**Pull request merged**